core/src/test/java/io/grpc/internal/ManagedChannelImplTest.java

```diff
@@ -4476,6 +4476,88 @@ public String getDefaultScheme() {
     }
   }
 
+  @Test
+  public void testApplyDefaultServiceConfigIfInitialNameResolutionFails() throws Exception {
+    LoadBalancerRegistry.getDefaultRegistry().register(mockLoadBalancerProvider);
+    timer.forwardNanos(1234);
+    channelBuilder.maxTraceEvents(10);
+    try {
+      Status resolutionError = Status.UNAVAILABLE
+              .withDescription("Initial Name Resolution error, using default service config");
+      FakeNameResolverFactory nameResolverFactory = new FakeNameResolverFactory
+              .Builder(expectedUri).setServers(Collections.singletonList(new
+              EquivalentAddressGroup(socketAddress))).setError(resolutionError).build();
+
```

Comment:
Remove unnecessary blank lines. For tests we try to follow setup, act, assert blocks with spaces only between these blocks.

Reply:
Fixed

Reply:
I still see blank lines at line 4491, 4496 etc. As per setup-act-assert block structure, there should only be blank lines above and below 
`nameResolverFactory.resolvers.get(0).listener.onError(resolutionError);`
for this test. Unnecessary blank lines make it a little difficult to quickly understand what the test is doing.

Reply:
Fixed